packages/translator/src/Charcoal/Translator/Translation.php

```diff
@@ -250,4 +252,15 @@ private function setVal($val)
 
         return $this;
     }
+
+    public function setIsRendered(bool $isRendered = true): self
+    {
+        $this->isRendered = $isRendered;
+        return $this;
+    }
+
+    public function isRendered(): bool
+    {
+        return isset($this->isRendered) && $this->isRendered === true;
+    }
```

Comment:
Maybe we could compact this into a single method?


```suggestion
    public function isRendered(?bool $rendered = null): bool
    {
        if (is_bool($rendered)) {
            $this->isRendered = $rendered;
        }

        return $this->isRendered;
    }
```